plugins/by-name/vim-suda/default.nix

```
@@ -0,0 +1,39 @@
+{ lib, helpers, ... }:
+
+lib.nixvim.vim-plugin.mkVimPlugin {
+  name = "vim-suda";
+  globalPrefix = "suda#";
+  maintainers = [ lib.maintainers.marcel ];
+
+  settingsOptions = {
+    executable = helpers.defaultNullOpts.mkStr "sudo" ''
+      Path to the sudo binary.
+    '';
+
+    noninteractive = helpers.defaultNullOpts.mkBool false ''
+      If set, suda will not prompt you for a password before saving a file.
+      It is supposed to support a setup with passwordless sudo or doas.
+      Use with care.
+    '';
+
+    prompt = helpers.defaultNullOpts.mkStr "Password: " ''
+      A prompt string used to ask password.
+    '';
+
+    smart_edit = helpers.defaultNullOpts.mkBool false ''
+      If set, an |autocmd| is created that performs a heuristic check on
```

We can use markdown in descriptions, so probably best to wrap `` `|doclinks` `` in backticks?

```suggestion
      If set, an `|autocmd|` is created that performs a heuristic check on
```

Also: if you want a paragraph break, you need to have an empty/blank line between paragraphs; single line-breaks are ignored by markdown :slightly_smiling_face:

---

As there are no paragraph breaks in the upstream docs, I'm just going not to insert some.